opentreemap/treemap/migrations/0032_rename_to_role_default_permission_level.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('treemap', '0031_add_custom_id_to_default_search_fields'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='role',
+            old_name='default_permission',
+            new_name='default_permission_level',
+        ),
+    ]
```

Comment:
Turns out you need a little dance to make this kind of migration -- the default "auto" migration deletes one field and creates another rather than renaming the existing one. The workaround suggested in https://code.djangoproject.com/ticket/24260 worked for me -- declare the `db_column` on the existing variable, make migrations, rename the variable, make migrations again, and then combine the two migrations.